channels/us.m3u

```diff
@@ -4549,13 +4549,13 @@ https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5f4796368174910007
 https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5f4796368174910007756454/master.m3u8?deviceId=channel&deviceModel=web&deviceVersion=1.0&appVersion=1.0&deviceType=rokuChannel&deviceMake=rokuChannel&deviceDNT=1&advertisingId=channel&embedPartner=rokuChannel&appName=rokuchannel&is_lat=1&bmodel=bm1&content=channel&platform=web&tags=ROKU_CONTENT_TAGS&coppa=false&content_type=livefeed&rdid=channel&genre=ROKU_ADS_CONTENT_GENRE&content_rating=ROKU_ADS_CONTENT_RATING&studio_id=viacom&channel_id=channel
 #EXTINF:-1 tvg-id="PlutoTVNickJr.us" tvg-name="Pluto TV Nick Jr." tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5ca6748a37b88b269472dad9/colorLogoPNG.png" group-title="Kids",Pluto TV Nick Jr.
 https://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5ede45451dce190007ef9ff2/master.m3u8?terminate=false&deviceType=web&deviceMake=web&deviceModel=web&sid=d1d4cf70-0714-11eb-a59f-0242ac110002&deviceId=5ede45451dce190007ef9ff2&deviceVersion=DNT&appVersion=DNT&deviceDNT=1&userId=&advertisingId=&deviceLat=&deviceLon=&app_name=&appName=web&buildVersion=&appStoreUrl=&architecture=&includeExtendedEvents=false&marketingRegion=DE&serverSideAds=false
-#EXTINF:-1 tvg-id="PlutoTVNickJrClub.us" tvg-name="Pluto TV Nick Jr. Club" tvg-country="US" tvg-language="Spanish" tvg-logo="https://images.pluto.tv/channels/5f121460b73ac6000719fbaf/colorLogoPNG.png" group-title="Kids",Pluto TV Nick Jr. Club
+#EXTINF:-1 tvg-id="PlutoTVNickJrClub.us" tvg-name="Pluto TV Nick Jr. Club" tvg-country="US" tvg-language="" tvg-logo="https://images.pluto.tv/channels/5f121460b73ac6000719fbaf/colorLogoPNG.png" group-title="Kids",Pluto TV Nick Jr. Club
```

Comment:
I think it isn't wrong that it is Spanish
In October of 2019, a Latino/Spanish version of the channel was launched. Further, in April 2020, a Latin American feed of Nick Jr. Pluto TV was launched. Known as Nick Jr. Club, the network's logo is in the shape of a cloud with "Nick Jr." in it on a light blue background.
https://nickelodeon.fandom.com/wiki/Nick_Pluto_TV#Nick_Jr._Pluto_TV

Reply:
I know they have a Latino version, but I thought they have Swedish, Russian, Portuguese also. Doesn't sound traditional Spanish. I'm in the Southwest, and it sounds more Russian to me. That's what I took it out until I can confirm what language that is. I will follow up.

Reply:
I can put it Spanish back in, if you all like, until I know for sure.

Reply:
I changed my mind. From the info I found some of Pluto Channels are intended for Brazil and Google Translate speech translation reacts more to Portuguese language

Reply:
So you want it as Portuguese?